### PR TITLE
[Help Editor] Module now a module

### PR DESCRIPTION
**Pull request by xlecours**
namespace
filenames
help content

### CODE REVIEW

File: modules/help_editor/help/help_editor.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Help Editor
+
+The Help Editor module displays existing help content for LORIS modules and pages. The main data table displays help content by Topic (Module or subpage), Parent Topic (Module or parent page) and Content. Use the selection filters to search by Help Topic or Search keyword. Keyword search will many any content in either the Topic or Content field. Sort the table by clicking on any column header.  
+
+** Editing Help Content **
```
Comment (driusan) [inline modules/help_editor/help/help_editor.md]: Semantically, shouldn't this be a second level header?

Comment (driusan) [reply modules/help_editor/help/help_editor.md]: (starting with "##")

File: modules/help_editor/php/module.class.inc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * This serves as a hint to LORIS that this module is a real module.
+ * It does nothing but implement the module class in the module's namespace.
+ *
+ * PHP Version 5
+ *
+ * @category Behavioural
+ * @package  Main
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris-Trunk/
```
Comment (driusan) [inline modules/help_editor/php/module.class.inc]: This URL is wrong (I suspect this isn't the only place, but I just noticed it now.. we can probably let it slide and then do a batch cleanup with sed after all this cleanup is done.)

Comment (xlecours) [reply modules/help_editor/php/module.class.inc]: Counts in the /modules firectories :
      6 https://www.github.com/aces/CCNA/
      7 https://www.github.com/aces/Loris
     45 https://www.github.com/aces/Loris/
     77 https://www.github.com/aces/Loris-Trunk/

I will fix this one here and the other in a second PR

**Comment by driusan** [comment]
I get the following error when trying to access the help on a page that hasn't been written in markdown on this branch:

```
<br />
<font size='1'><table class='xdebug-error xe-uncaught-exception' dir='ltr' border='1' cellspacing='0' cellpadding='1'>
<tr><th align='left' bgcolor='#f57900' colspan="5"><span style='background-color: #cc0000; color: #fce94f; font-size: x-large;'>( ! )</span> Fatal error: Uncaught Error: Class 'HelpFile' not found in /var/www/loris/modules/help_editor/ajax/help.php on line <i>42</i></th></tr>
<tr><th align='left' bgcolor='#f57900' colspan="5"><span style='background-color: #cc0000; color: #fce94f; font-size: x-large;'>( ! )</span> Error: Class 'HelpFile' not found in /var/www/loris/modules/help_editor/ajax/help.php on line <i>42</i></th></tr>
<tr><th align='left' bgcolor='#e9b96e' colspan='5'>Call Stack</th></tr>
<tr><th align='center' bgcolor='#eeeeec'>#</th><th align='left' bgcolor='#eeeeec'>Time</th><th align='left' bgcolor='#eeeeec'>Memory</th><th align='left' bgcolor='#eeeeec'>Function</th><th align='left' bgcolor='#eeeeec'>Location</th></tr>
<tr><td bgcolor='#eeeeec' align='center'>1</td><td bgcolor='#eeeeec' align='center'>0.0004</td><td bgcolor='#eeeeec' align='right'>360632</td><td bgcolor='#eeeeec'>{main}(  )</td><td title='/var/www/loris/htdocs/AjaxHelper.php' bgcolor='#eeeeec'>.../AjaxHelper.php<b>:</b>0</td></tr>
<tr><td bgcolor='#eeeeec' align='center'>2</td><td bgcolor='#eeeeec' align='center'>0.0480</td><td bgcolor='#eeeeec' align='right'>1063048</td><td bgcolor='#eeeeec'>require( <font color='#00bb00'>'/var/www/loris/modules/help_editor/ajax/help.php'</font> )</td><td title='/var/www/loris/htdocs/AjaxHelper.php' bgcolor='#eeeeec'>.../AjaxHelper.php<b>:</b>93</td></tr>
</table></font>
```

**Comment by xlecours** [comment]
@driusan Please test again

**Comment by driusan** [comment]
The styling on the preview is wrong, but it's also wrong in the same way on aces/minor, so it looks like this PR is working now.

**Pull request merged**